Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -1565,7 +1565,10 @@ RefPtr<Font> FontCache::systemFallbackForCharacters(const FontDescription& descr
 
     auto [syntheticBold, syntheticOblique] = computeNecessarySynthesis(substituteFont, description, ShouldComputePhysicalTraits::No, isForPlatformFont == IsForPlatformFont::Yes).boldObliquePair();
 
-    FontPlatformData alternateFont(substituteFont, platformData.size(), syntheticBold, syntheticOblique, platformData.orientation(), platformData.widthVariant(), platformData.textRenderingMode(), platformData.creationData());
+    const FontPlatformData::CreationData* creationData = nullptr;
+    if (safeCFEqual(platformData.font(), substituteFont))
+        creationData = platformData.creationData();
```

Comment:
Can we assert in here that if `platformData` is not `substituteFont`, then it's an installed font?

Reply:
Hrmmm I don't see a way to do that 🤔

Reply:
I'll land as-is and in parallel discuss with the Core Text team.